src/anchor.cpp

```diff
@@ -170,6 +172,7 @@ void Anchor::Walk() {
   }
   double dr = step_direction_ * vel * delta_;
   mesh_lambda_ += dr;
+  // Should this also add to bond lambda?
```

Comment:
No, the bond lambda gets updated only when forces need to be calculated on the bond. Which is confusing, so I had to look into the code to answer this question. Maybe that should be changed?